include/ck_tile/ops/fmha/kernel/fmha_fwd_splitkv_kernel.hpp

```diff
@@ -628,21 +601,32 @@ struct FmhaFwdSplitKVKernel
                 k_dram_naive,
                 make_tuple(number<FmhaPipeline::kN0>{}, number<FmhaPipeline::kK0>{}),
                 sequence<kPadSeqLenK, kPadHeadDimQ>{});
+        };
+        const auto k_dram = [&]() {
+            if constexpr(kIsPagedKV)
+            {
+                return make_k_dram(nullptr, kargs.page_block_size);
```

Comment:
Hi, Does this mean create a global memory tensor/tile on pointer with shape (`page_block_size`, `hdim_q`), stride (`stride_k`, `1`)? And what doese the `will update this pointer if using paged-kvcache` mean?  Does it mean this memory tensor will move to next k/v cache page?
Thanks for replying!

Reply:
Yes, we use tensor view here to describe the global memory (shape & stride). When moving to next k/v cache page, we will have to update underlying data pointer to point to the correct page.

Reply:
Thanks for you kind explanation. I am working on AMD gpus and occasionally find this PR, trying to learn the ck_tile.
So from my point of view, creating `k_ptrs` and `v_ptrs` could be unnecessary when the paged kv is enabled?
```
        const KDataType* k_ptr =
            reinterpret_cast<const KDataType*>(kargs.k_ptr) +
            static_cast<long_index_t>(i_nhead / kargs.nhead_ratio_qk) * kargs.nhead_stride_k +
            batch_offset_k;
        const VDataType* v_ptr =
            reinterpret_cast<const VDataType*>(kargs.v_ptr) +
            static_cast<long_index_t>(i_nhead / kargs.nhead_ratio_qk) * kargs.nhead_stride_v +
            batch_offset_v;
```

Reply:
Yes, both of the `k_ptr` & `v_tpr` are only for non-paged implementations. Offset computations for paged k/v cache is in the navigator and the navigator creation lambdas.